src/subdomains/supporting/payment/entities/transaction-specification.entity.ts

```diff
@@ -24,11 +24,15 @@ export class TransactionSpecification extends IEntity {
   @Column({ type: 'float' })
   minFee: number; // CHF
 
+  @Column({ nullable: true })
+  minConfirmations?: number;
+
   static default(): TransactionSpecification {
     const spec = new TransactionSpecification();
 
     spec.minVolume = 0;
     spec.minFee = 0;
+    spec.minConfirmations = 1;
 
```

Comment:
Seems like we use `tx.confirmations > minConfirmations` everywhere -> the default would probably be 0. Or we change the comparison to `>=`.